app/routes/signin-oidc.js

```diff
@@ -134,27 +142,20 @@ module.exports = [{
 
         await cphCheck.customerMustHaveAtLeastOneValidCph(request, apimAccessToken)
 
-        const endemicsApplyJourney = `${config.applyServiceUri}/endemics/check-details`
-        const oldClaimJourney = `${config.claimServiceUri}/signin-oidc?state=${request.query.state}&code=${request.query.code}`
         const latestApplicationsForSbi = await applicationApi.getLatestApplicationsBySbi(organisation.sbi, request.logger)
-        const applyRedirectionLink = sendToApplyJourney(latestApplicationsForSbi, loginSource, h, endemicsApplyJourney, organisation)
-
-        if (typeof applyRedirectionLink === 'string') {
-          return h.redirect(applyRedirectionLink)
-        }
+        const redirectPath = sendToApplyJourney(latestApplicationsForSbi, loginSource, organisation)
 
-        // they have an open old world application/claim
-        if (loginSource === loginSources.apply) {
```

Comment:
I assume this was removed, along with the error it might throw because it can't ever be reached, given sendToApplyJourney(...) always either returns a string or throws. is that right?

Reply:
Yes there's no actual way to get here anymore, you would always have been redirected just before hand.

Reply:
I'll flag this one with @berlin-ee and Prab to be aware of for any regression testing though.